app/views/streams/_job_status.html.erb

```diff
@@ -13,8 +16,9 @@
             </span>
           </button>
         </h4>
-        <div id="collapse<%= status_group.to_s.classify %>" class="accordion-collapse collapse"
+        <div id="collapse<%= status_group.to_s.classify %>" class="accordion-collapse collapse <%= !tab_expanded && jobs.count > 0 ? 'show' : 'collapsed' %>"
```

Comment:
Since you're going to need to use this logic in more than one place maybe it would make sense to write a helper method? It could handle the logic needed to support only opening one accordian even if more than one has jobs, etc.